conf/gulp-tasks/library.gulpfile.js

```diff
@@ -111,11 +113,34 @@ async function createBundleTaskFactory (locale) {
 }
 
 function compileCSS () {
+  return new Promise(resolve => {
+    return parallel(compileLTRCSS, compileRTLCSS)(resolve);
+  });
+}
+
+/**
+ * Compiles the standard left to right CSS
+ */
+function compileLTRCSS () {
```

Comment:
Maybe LTR and RTL shouldn't be all caps? since right now `LTRCSS` is all mushed together

Reply:
sure, i'll update to `compileLtrCss` and `compileRtlCss`